### ISSUE TITLE
Code still work?

### ISSUE THREAD

**Issue by trangtiu** [cassiompf/lol-authenticator#3]
Does this code still work? I tried downloading it and it doesn't work anymore

**Comment by QuantumKuba** [comment]
Can you specify what the issue is? 

Are there any errors showing?

**Comment by cassiompf** [comment]
> Does this code still work? I tried downloading it and it doesn't work anymore

Yes, it works. What's going on for you? Does an error appear? If yes, what error appears?

**Comment by karimawi** [comment]
> Yes, it works. What's going on for you? Does an error appear? If yes, what error appears?

I keep getting this:
```{'errorCode': 'RPC_ERROR', 'httpStatus': 513, 'implementationDetails': {}, 'message': 'Malformed request'}```

It was working okay before, idk what happened.

**Comment by cassiompf** [comment]
> > Yes, it works. What's going on for you? Does an error appear? If yes, what error appears?
> 
> I keep getting this: `{'errorCode': 'RPC_ERROR', 'httpStatus': 513, 'implementationDetails': {}, 'message': 'Malformed request'}`
> 
> It was working okay before, idk what happened.

I think the Riot client got some update. I'll try to make this update in the repository so that it works again.

**Comment by trangtiu** [comment]
@cassiompf  hope you will continue to develop it
